### PR TITLE
Load test post priority feature

### PR DESCRIPTION
**Pull request by koox00**
#### Summary

This PR is created to be able to load test Mattermost with PostPrioirty feature flag enabled.

Adds a config option `PercentUrgentPosts` for use in both `gencontroller` and `simulcontroller` which would be the percentage of posts created that will have the "urgent" priority.
Also enables the PostPriority feature flag, and ServiceSetting.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-46410

### CODE REVIEW

File: loadtest/control/gencontroller/actions.go [hunk 1]
```diff
@@ -189,11 +189,23 @@ func (c *GenController) createPost(u user.User) control.UserActionResponse {
...
 
-	postId, err := u.CreatePost(&model.Post{
+	post := &model.Post{
 		Message:   control.GenerateRandomSentences(wordCount) + channelMention,
 		ChannelId: channel.Id,
 		CreateAt:  time.Now().Unix() * 1000,
-	})
+	}
+
+	// 5% of the times post will have urgent priority.
```
Comment (agnivade) [inline loadtest/control/gencontroller/actions.go:198]: Where is this number coming from? Have we ran some queries on community to check this?

**Comment by agarciamontoro** [comment]
Hey, @koox00, is this ready for review?

**Comment by agnivade** [comment]
Gentle ping on this @koox00

**Comment by agnivade** [comment]
Gentle reminder @koox00

**Comment by agarciamontoro** [comment]
Synced with @koox00 and now that the feature is live on Community, he'll take a look at real numbers to come up with a realistic frequency. There's also the last part of the feature pending, but I guess we can review this as soon as this part is finished and open a different PR updating the code with that last part. Does that sound good, @koox00?

**Comment by koox00** [comment]
yep that sounds good. sorry for not communicating this earlier.
thanks @agarciamontoro

**Comment by agarciamontoro** [comment]
@koox00, no worries. Just ping us here when we can review this :)

**Review by agarciamontoro — APPROVED** [review]
This looks great! Have you had a chance to look at real data to adjust the percentage number?

**Comment by koox00** [comment]
@agnivade, @agarciamontoro updated the PR with metrics from the community server.
The actual percentage of urgent posts in community is 0.001, 
I've made it 10 times that.

File: deployment/terraform/strings.go [hunk 2]
```diff
@@ -17,6 +17,7 @@ WorkingDirectory=/opt/mattermost
 User=ubuntu
 Group=ubuntu
 LimitNOFILE=49152
+Environment=MM_FEATUREFLAGS_POSTPRIORITY=true
```
[the remark this answers is not in the document]
Comment (agnivade) [reply deployment/terraform/strings.go:20]: Sure

File: loadtest/config.go [hunk 3]
```diff
@@ -95,6 +95,8 @@ type InstanceConfiguration struct {
 	PercentReplies float64 `default:"0.5" validate:"range:[0,1]"`
 	// The percentage of replies that should be in long threads
 	PercentRepliesInLongThreads float64 `default:"0.05" validate:"range:[0,1]"`
+	// The percentage of post that are marked as urgent
+	PercentUrgentPosts float64 `default:"0.05" validate:"range:[0,1]"`
```
Comment (agnivade) [inline loadtest/config.go:99]: Why is this different from 0.001?

**Review by agnivade — APPROVED** [review]
Left a nonblocking comment

File: loadtest/config.go (hunk 3 above)
Comment (koox00) [reply loadtest/config.go:99]: oh, I thought I change all occurrences, thanks Agniva will change

**Comment by agarciamontoro** [comment]
Yes!!! :muscle::muscle::muscle:  Thank you for your efforts here, @koox00 <3

**Pull request merged**